README.md

```diff
@@ -88,6 +88,15 @@ Bokeh
     </a>
   </td>
 </tr>
+<tr>
+	<td>AppVeyor</td>
+	<td>
+	<a href="https://ci.appveyor.com/project/xavArtley/bokeh">
+		<img src=https://ci.appveyor.com/api/projects/status/github/xavArtley/bokeh
+			alt="Current AppVeyor build status" />
+	</a>
+	</td>
+</tr>
```

Comment:
We will need to make a bokeh project account and have the link here refer to that. For now can you just leave this out? I need to re-edit the README anyway, I will re-organize that table to accommodate this badge at that point.

Reply:
OK we already have a bokeh project on Appveyor, linked under the "bokeh-integrations" GitHub account. If you let me know any specific configuration necessary, I can make any changes.

Reply:
I just sign in on appveyor with my github account and add the repo to projects. Now each time a modification is done its trigger a build